packages/task/src/common/task-protocol.ts

```diff
@@ -57,6 +57,8 @@ export interface TaskConfiguration extends TaskCustomization {
      * This field is not supposed to be used in `tasks.json`
      */
     readonly _scope: string | undefined;
+    /** Whether the task is a background task or not. */
+    isBackground?: boolean;
```

Comment:
If `isBackground` can be used to mark the detected tasks as background tasks, we might want to add `isBackground` to `interface TaskCustomization`.

Reply:
Also, do we want to add `dependsOrder` and `dependsOn` to this interface as well?

Reply:
fixed, done !